ls_0/index.html

```diff
@@ -0,0 +1,380 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="/ls_0/style.css">
+    <title>Document</title>
+</head>
+
+<body>
+    <div class="container">
+
+        <header>
+            <nav>
+                <div class="nav-main">
+                    <div class="nav-main-left"><img src="/ls_0/src/images/Union.svg" alt="logo"></div>
+                    <div class="nav-main-right">Explore</div>
+                    <div class="nav-menu">
```

Comment:
This is the list of nav items. So it`s more convinient to use more apropriate (semantic) tags for this.